roles/openshift_manageiq/vars/main.yml

```diff
@@ -27,6 +27,7 @@ manage_iq_tasks:
     - policy add-role-to-user -n management-infra admin -z management-admin
     - policy add-role-to-user -n management-infra management-infra-admin -z management-admin
     - policy add-cluster-role-to-user cluster-reader system:serviceaccount:management-infra:management-admin
+    - policy add-cluster-role-to-user image-auditor system:serviceaccount:management-infra:management-admin
```

Comment:
> This role will need to be conditionalized on openshift.common.version_gte_3_2_or_1_2 since the role doesn't exist prior to that.

@detiber During testing the upgrade scenario I installed with ```openshift_pkg_version=-1.2.0``` and found out the role does not exist there(with latest ):

[root@mtayer-centos7-4 ~]# oc version
oc v1.2.0-rc1
kubernetes v1.2.0-36-g4a3f9c5
[root@mtayer-centos7-4 ~]# oc get clusterrole/system:image-auditor
Error from server: role "system:image-auditor" not found

I can not explain this since when I installed regularly (not upgrade) there was no problem